code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm

```diff
@@ -0,0 +1,39 @@
+
+// --------------------------------------------------------------------------------
+// ----------------- TERROR SPIDERS: T2 BLACK TERROR ------------------------------
+// --------------------------------------------------------------------------------
+// -------------: ROLE: assassin, poisoner, DoT expert
+// -------------: AI: attacks to inject its venom, then retreats. Will inject its enemies multiple times then hang back to ensure they die.
+// -------------: SPECIAL: venom that does more damage the more of it is in you
+// -------------: TO FIGHT IT: if bitten once, retreat, get charcoal/etc treatment, and come back with a gun.
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/GradualGrinder
```

Comment:
The fact it's referencing something like TV tropes come across as memetic and cringey, if I'm honest. It's like citing wikipedia in the middle of a research paper. It might not necessarily be *wrong*, but it's still kinda "ehhhhh..".